scipy/optimize/tests/test_linprog.py

```diff
@@ -876,10 +888,18 @@ def test_bug_5400(self):
         if self.method == 'simplex':
             with pytest.warns(OptimizeWarning):
                 res = linprog(c, A_ub, b_ub, bounds=bounds,
-                    method=self.method, options=self.options)
-        else:
+                              method=self.method, options=self.options)
```

Comment:
Here and below the behaviour for ``simplex`` and ``revised-simplex`` vary for the same issue. ``simplex`` returns ``status=0`` (problem solved successfully) and raises the warning message. I'm not sure if ``status=4`` is the best option. In this instance the returned result is still optimal  and is resolved despite the numerical issues. Wouldn't this make ``0`` a better status?

Reply:
I think this is fixed so that status = 4 won't happen anymore.